packages/document-model-libs/editors/chat-room-editor/editor.tsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable react/jsx-no-bind */
+import { EditorProps } from "document-model/document";
+import {
+  ChatRoomState,
+  ChatRoomAction,
+  ChatRoomLocalState,
+  ReactionType,
+  actions,
+} from "../../document-models/chat-room";
```

Comment:
with the typescript fix, you should now be able to just import these from `document-models/chat-room` instead of needing the relative path. This is most likely what your editor wants to do already